modules/support-collecting-network-trace.adoc

```diff
@@ -60,7 +60,7 @@ $ oc debug node/my-cluster-node
 +
 [NOTE]
 ====
-If an existing `toolbox` Pod is already running, the `toolbox` command outputs `'toolbox-' already exists. Trying to start...`. To avoid `tcpdump` issues, remove the running toolbox container with `podman rm toolbox-` and spawn a new toolbox container.
+If an existing `toolbox` pod is already running, the `toolbox` command outputs `'toolbox-' already exists. Trying to start...`. To avoid `tcpdump` issues, remove the running toolbox container with `podman rm toolbox-` and spawn a new toolbox container.
```

Comment:
I'd be surprised if it actually outputs `toolbox-` rather than `toolbox-<something>`. I wonder if this text is accurate?